client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -62,6 +64,30 @@ class SiteSettingsFormDiscussion extends Component {
 		);
 	}
 
+	commentDisplaySettings() {
+		const { isJetpack, isCommentsModuleActive, jetpackVersionSupportsCalypsoSettingsUI } = this.props;
```

Comment:
Currently, if the Comments module is disabled, we'll hide these settings, but there's no way for the user to enable the module in Calypso. 

I'd suggest adding a toggle to enable/disable the Comments module above the settings. We have already taken care of all the Redux work related with module activation/deactivation, so you'd just have to connect the new `<CommentDisplaySettings />` component to have access to it. Feel free to borrow the approach for the toggle from #10083.

The current implementation also contradicts with the mockup for this card in p6TEKc-HQ-p2.